### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by luke780440**

### CODE REVIEW

**Comment by luke780440** [comment]
First iteration

File: AzureBot_Test.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+from threading import Thread
+import serial
+import controls
+import gui
+
+class Comms:
+    def __init__(self, port: int, baud_rate: int):
```
Comment (ayambabu23) [inline AzureBot_Test.py:7]: port is not an int, it's a string

File: AzureBot_Test.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        self.port = port
+        self.baud_rate = baud_rate
+        ser = serial.Serial(self.port, self,baud_rate)
+        self.ser.close()
```
Comment (ayambabu23) [inline AzureBot_Test.py:11]: Good practice, delete once you've seen it

File: AzureBot_Test.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            rightJoy_LR = packetControls.packet[2]
+            rightJoy_UD = packetControls.packet[3]
+            servoRotate = packetControls.packet[4]
+            servoGrab = packetControls.packet[5]
+
+            while True:
+                
+                #coding the Left thruster (sendSystemsLes)
+                
+                if leftJoy_LR < [43,43]:
```
Comment (ayambabu23) [inline AzureBot_Test.py:35]: Questionable line, what are you trying to do?

File: AzureBot_Test.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+                    value = send_value(leftJoy_LR)
```
Comment (ayambabu23) [inline AzureBot_Test.py:36]: self.send_value

Comment (ayambabu23) [reply AzureBot_Test.py:36]: Change for all

File: AzureBot_Test.py [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+                    packet_leftJoy_stopped = chr(1) + chr(7) + chr((self.value).encode("latin")) + chr(12)
+                    self.ser.write(packet_leftJoy_stopped)
+                else:
+                    value = send_value(leftJoy_LR)
+                    packet_leftJoy = chr(1) + chr(7) + chr((self.value).encode("latin")) + chr(12) 
+                    self.ser.write(packet_leftJoy)
+
+                #coding the Right thruster
+                if rightJoy_LR < [43,43]:
```
Comment (ayambabu23) [inline AzureBot_Test.py:45]: Again, questionable line

File: AzureBot_Test.py [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+                    value = send_value(rightJoy_LR)
+                    packet_rightJoy_stopped = chr(1) + chr(6) + chr((self.value).encode("latin")) + chr(12)
```
Comment (ayambabu23) [inline AzureBot_Test.py:47]: Do chr(0) instead

File: AzureBot_Test.py [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+                    self.ser.write(packet_rightJoy_stopped)
+                else:
+                    packet_rightJoy = chr(1) + chr(6) + chr((self.value).encode("latin")) + chr(12) 
+                    self.ser.write(packet_rightJoy)
+
+                #servo claw code - finalized with packet value
+                if servoRotate == True:
+                    packet_servoRotate = chr(1) + chr(10) + chr(12)
```
Comment (ayambabu23) [inline AzureBot_Test.py:55]: Do 4 bytes

File: AzureBot_Test.py [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+                if rightJoy_LR < [43,43]:
+                    value = send_value(rightJoy_LR)
+                    packet_rightJoy_stopped = chr(1) + chr(6) + chr((self.value).encode("latin")) + chr(12)
+                    self.ser.write(packet_rightJoy_stopped)
+                else:
+                    packet_rightJoy = chr(1) + chr(6) + chr((self.value).encode("latin")) + chr(12) 
+                    self.ser.write(packet_rightJoy)
+
+                #servo claw code - finalized with packet value
+                if servoRotate == True:
```
Comment (ayambabu23) [inline AzureBot_Test.py:54]: Treat servo like you would treat thruster; write code similarly: if the button is held, keep opening. If the button is released, stop opening. If you have questions, come and ask me.

File: AzureBot_Test.py [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+                #read data
+                #test and figure out what each byte means
+                #take data, convert it, and send to systems
+                #[-127,127]
+
+    def start_thread(self):
+        start_thread = Thread(target = self.run)  
+        start_thread.start()
+    
+    displayGUI = gui.GUI(#add info later)
```
Comment (ayambabu23) [inline AzureBot_Test.py:72]: Comment out this entire line until you've actually done something related to this

**Review by ayambabu23 — CHANGES_REQUESTED** [review]
Let me know if you have questions

File: AzureBot_Test.py [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+        value = round(value*1.27)
+        if value<0:
+            value = 255+value
+        return value 
+
+    def run(self):
+        while True: 
+            packetControls = controls.Controls() #change later
+            leftJoy_LR = packetControls.packet[0] #and repeat... will finalize once I know what values will come
+            leftJoy_UD = packetControl.packet[1]
```
Comment (luke780440) [inline AzureBot_Test.py:25]: I'm a bit confused about what you're trying to say about my code.

**Comment by luke780440** [comment]
@ayambabu23 

Please code review

**Pull request closed without merging**